metadata.yaml

```diff
@@ -23,7 +23,7 @@ containers:
 resources:
   agent-image:
     type: oci-image
-    upstream-source: grafana/agent:v0.20.1
+    upstream-source: grafana/agent:v0.26.1
```

Comment:
Update README as well?
What do you think about a resource revision table ([example](https://github.com/canonical/alertmanager-k8s-operator/blob/main/README.md#resource-revisions))?

Reply:
We should actually remove that from Alertmanager instead. Having a resource revision table is practically begging users to deploy mismatched charm+workload.

Reply:
I agree, it's asking for trouble.